torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2199,6 +2199,28 @@ def sample_inputs_masked_select(op_info, device, dtype, requires_grad):
                                 dtypes=[torch.bfloat16])),
                    safe_casts_outputs=True,
                    handles_complex_extremals=False),
+    UnaryUfuncInfo('square',
+                   ref=np.square,
+                   dtypes=all_types_and_complex_and(torch.bool),
+                   dtypesIfCPU=all_types_and_complex_and(torch.bool),
+                   dtypesIfCUDA=all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16),
+                   skips=(
+                       # Reference: https://github.com/pytorch/pytorch/issues/52549
+                       SkipInfo('TestUnaryUfuncs', 'test_reference_numerics',
```

Comment:
These skips will need an update because @zasdfgbnm just refactored this test into 3 parts, which may allow for better test coverage.